wagtail/models/reference_index.py

```diff
@@ -155,6 +155,12 @@ class ReferenceIndex(models.Model):
 
     wagtail_reference_index_ignore = True
 
+    # Store the models that are being tracked and indexed.
+    # If a model is present as a key, the model is tracked by receiving the corresponding
+    # model save and delete signals. If the value is True, then it is indexed.
+    # (False indicates the model has a ParentalKey relation with an indexed model.)
+    _indexed_models = {}
```

Comment:
I think it might be clearer to represent this as two sets: one for tracked models and one for indexed models.